cpu/BUILD

```diff
@@ -33,8 +33,8 @@ constraint_setting(name = "cpu")
 
 # TODO(b/136237408): Remove this generic CPU name and replace with a specific one.
 alias(
-    name = "aarch64",
-    actual = ":arm64",
+    name = "arm64",
+    actual = ":aarch64",
 )
 
 # TODO(b/136237408): Remove this generic CPU name and replace with a specific one.
```

Comment:
Perhaps we should add aarch32 as fundamental type at the same time.
"arm" can be an alias of that.